### PR TITLE
Added the first draft of README.md

### PR DESCRIPTION
**Pull request by halfzebra**
@raulpineda I have added the first draft, WIP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+- Rendering HTML and CSS
+- Building interactive applications
+
+## Introduction
+
+Welcome to Codher's Elm Workshop!
+
+Functional Programming is fun and easy if you learn by doing.
+
+Have you ever tried programming in a language, where compiler is your best friend? Imagine an app without errors at run-time.
```
Comment (raulpineda) [inline README.md]: where the compiler

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+Sounds promising?
+
+Immutabile, Pure and Reactive by nature all that awailable now!
```
Comment (raulpineda) [inline README.md]: Maybe mention why are immutable, pure and reactive good to have characteristics. Like "pure functions by nature, so you don't have to worry about ugly and unexpected side effects"

Comment (raulpineda) [reply README.md]: available

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+
+Get an introduction to Elm Compiler and Time-Travelling Debugger and start coding!
```
Comment (raulpineda) [inline README.md]: In this workshop you'll get an introduction to the Elm Compiler (so you can build your first application or something), and the Time-Travelling debugger (so you...) and start coding in Elm!

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+
+Level: Basic understanding of JavaScript
+
+Happy coding, have fun!
+
+## Recommended Software
+
+You can use whichever code editor you prefer.
```
Comment (raulpineda) [inline README.md]: Maybe add suggestions? A link to atom or vs code or whatever

File: README.md [hunk 5]
```diff
@@ -1 +1,38 @@
...
+- Getting started with Elm and Create Elm App
+- Intro into Functional Programming
+- Rendering HTML and CSS
+- Building interactive applications
+
+## Introduction
+
+Welcome to Codher's Elm Workshop!
+
+Functional Programming is fun and easy if you learn by doing.
```
Comment (raulpineda) [inline README.md:7]: Maybe start with the benefits of functional programming (the immutable, pure and reactive paragraph) and then say it's easy and fun

File: README.md [hunk 6]
```diff
@@ -1 +1,38 @@
-# elm-workshop
\ No newline at end of file
+# Codher Purely Functional Interfaces in Elm
+
+## Agenda
```
Comment (raulpineda) [inline README.md]: The agenda should go after the introduction

**Review by raulpineda — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (coquer) [inline README.md]: Where the compiler

File: README.md (hunk 2 above)
Comment (coquer) [inline README.md]: Immutable

**Comment by halfzebra** [comment]
@raulpineda @jycr753 Thanks for the review! I have fixed the typos and included more info.

**Review by raulpineda — APPROVED** [review]

**Comment by raulpineda** [comment]
Looks great! Let me know if it's still a WIP, otherwise I'll merge :)

**Comment by halfzebra** [comment]
@raulpineda Thanks!

I think that's pretty much it for the description, I'll start working on examples and exercises.

Feel free to merge this PR! :slightly_smiling_face:

**Pull request merged**